modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -96,6 +103,19 @@ Because you create and manage the worker machines yourself, you do not need
 to initialize these machines.
 endif::aws,gcp[]
 
+ifdef::osp[]
```

Comment:
I'm not sure if the rendered doc is outdated[1], but this removal is not present in the Kuryr section:

[1] http://file.rdu.redhat.com/mbridges/031320/shiftstack-upi-OSDOCS921/installing/installing_openstack/installing-openstack-user-kuryr.html#installation-user-infra-generate-k8s-manifest-ignition_installing-openstack-user-kuryr

Reply:
@MaysaMacedo Nice catch! This was just a bad conditional statement in the docs. Should be fixed, now: 

- Kuryr: http://file.rdu.redhat.com/mbridges/031620/shiftstack-upi-OSDOCS921/installing/installing_openstack/installing-openstack-user-kuryr.html#installation-user-infra-generate-k8s-manifest-ignition_installing-openstack-user-kuryr
- Default: http://file.rdu.redhat.com/mbridges/031620/shiftstack-upi-OSDOCS921/installing/installing_openstack/installing-openstack-user.html#installation-user-infra-generate-k8s-manifest-ignition_installing-openstack-user

(Second link is a bit of assumption, as file.rdu.redhat.com is now down for me, but the change looks good locally.)